horovod/common/__init__.py

```diff
@@ -37,11 +38,17 @@ def get_ext_suffix():
                              'mpi_lib' + get_ext_suffix()), mode=ctypes.RTLD_GLOBAL)
 
 
-def init():
+def init(mpi_init=True ,mpi_finalize=True):
```

Comment:
Please run `autopep8` and `clang-format` for Python and C++ source files that you change.